plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/get_alert_evidence/schema.py

```diff
@@ -0,0 +1,174 @@
+# GENERATED BY INSIGHT-PLUGIN - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Allows the user to retrieve evidence for a single alert from an alert RRN"
+
+
+class Input:
+    ALERT_RRN = "alert_rrn"
+    INDEX = "index"
+    SIZE = "size"
```

Comment:
Same here :)

Reply:
yes good catch, they needed moved after an update and looks like they were not re-added correctly, i'l update to add them back in again